package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Digital asset management project",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint ./src",
+    "prettier": "prettier './**/*.{js,jsx,json,html}' --write"
```

Comment:
This adds a command to run Prettier on our codebase. So instead of running that big command every time we can simple run `npm run prettier` in the command line.
The pattern with the asterisk is a "glob pattern". If you're not familiar with it, it's a good thing to look up! But it's basically a way to denote patterns of files. So in this case
- `.` means: current directory
- `/**` means: any level of directory nesting (no nesting to infinite nesting)
- `/*.{js,jsx,json,html}` means: any file that has any name and ends with a `.` and any of those file extensions`